pages/linux/conky.md

```diff
@@ -0,0 +1,27 @@
+# conky
+
+> Light-weight system monitor for X.
+
+- Launch with default, built-in config:
+
+`conky`
+
+- Create a new, default config:
+
+`conky -C > ~/.conkyrc`
+
+- Load a config file:
+
+`conky -c {{/path/to/config}}`
+
+- Start in the background (daemonize):
+
+`conky -d`
+
+- Align specified config file on screen:
+
+`conky -c {{path/to/config}} -a {{{top,bottom,middle}_{left,right,middle}}}`
+
+- Wait 5 seconds and launch the conky config in a specified path:
+
+`conky -p 5 -c {{path/to/config}}`
```

Comment:
What does `-p` stand for ? Can we use a word in the description that matches with that ?. And no need to use the `-c` option again. It is already specified above. And users are expected to combine options.

Reply:
the `-p` option is to pause for {SECS} seconds at startup before launching. I will change it to that I think.